docs/specs/xref.yml

```diff
@@ -563,3 +563,19 @@ inputs:
 outputs:
   docs/a.json:
   docs/b.json:
+---
+# Mix cyclic uid reference with markdown inclusion
+inputs:
+  docfx.yml:
+  docs/a.md: |
+    @b?displayProperty=summary
+  docs/b.json: |
+    {
+      "$schema": "https://raw.githubusercontent.com/dotnet/docfx/v3/schemas/TestData.json",
+      "uid": "b",
+      "summary": "[!include[](a.md)]",
+    }
+outputs:
+  build.log: |
+    ["error","circular-reference","Found circular reference: 'docs/a.md' --> 'docs/a.md'","docs/b.json"]
+    ["error","circular-reference","Found circular reference: 'docs/a.md' --> 'docs/a.md'","docs/a.md"]
```

Comment:
this is what i mean, we error message is missing b.json

Reply:
There should be some bugs in xref, I'm working on that.